### PR TITLE
Add Prow jobs for kubevirt/bridge-maker

### PR DESCRIPTION
**Pull request by danielBelenky**
Signed-off-by: Daniel Belenky <dbelenky@redhat.com>

### CODE REVIEW

**Comment by danielBelenky** [comment]
@phoracek I've had to rebase this one, could you review again, please?

File: github/ci/prow/files/jobs/bridge-maker/bridge-maker-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+presubmits:
+  nmstate/kubernetes-nmstate:
```
Comment (qinqon) [inline github/ci/prow/files/jobs/bridge-maker/bridge-maker-presubmits.yaml]: This is not for nmstate

**Pull request merged**